JavaScript/JavaScriptSDK/Telemetry/RemoteDependencyData.ts

```diff
@@ -0,0 +1,46 @@
+﻿/// <reference path="../Contracts/Generated/PageViewData.ts" />
+/// <reference path="./Common/DataSanitizer.ts"/>
+/// <reference path="../Contracts/Generated/RemoteDependencyData.ts"/>
+
+module Microsoft.ApplicationInsights.Telemetry {
+    "use strict";
+
+    export class RemoteDependencyData extends AI.RemoteDependencyData implements ISerializable {
+
+        public static envelopeType = "Microsoft.ApplicationInsights.RemoteDependencyData";
+        public static dataType = "RemoteDependencyData";
+
+        public aiDataContract = {
```

Comment:
This is the data class I construct inside appInsights.trackAjax() - https://github.com/Microsoft/ApplicationInsights-JS/blob/ajax_proto/JavaScript/JavaScriptSDK/appInsights.ts#L282